health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/ImpelComponentWrapper.js

```
@@ -0,0 +1,187 @@
+
+    const addingValidations = (field) => {
+      switch (field.type) {
+        case "number":
+          if (!field.min && !field.max) return {};
+          return {
+            validation: {
+              min: field.min,
+              max: field.max,
+            },
+          };
+          break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)